docs/content/topics/administrators/authentication.md

```diff
@@ -11,4 +11,77 @@ aliases:
 
 # Brigade Authentication
 
-TODO
+Brigade utilizes third-party providers for its user authentication strategy.
```

Comment:
Do you think it's worthwhile to explain the rationale for this? I could go either way.

The rationale is that with Brigade 2 providing an abstraction between users and the underlying substrate (Kubernetes), it cannot be taken for granted that every user of Brigade is also a Kubernetes user. That introduces a need for sourcing user identity from somewhere else.

Reply:
Good call, I'd been taking that point for granted, definitely worth a mention.  Will add.

Reply:
My wording was quick and dirty tho. I'm sure you can improve on it.

Reply:
Ok, updated.  PTAL when convenient.  Thanks!